src/Parser/Hunter/Marksmanship/Modules/Items/Tier20_2p.js

```diff
@@ -4,26 +4,37 @@ import Combatants from 'Parser/Core/Modules/Combatants';
 import SPELLS from 'common/SPELLS';
 import SpellIcon from 'common/SpellIcon';
 import SpellLink from 'common/SpellLink';
-import { formatPercentage } from "common/format";
+import HIT_TYPES from "Parser/Core/HIT_TYPES";
+import { formatPercentage, formatNumber} from "common/format";
+import getDamageBonus from "../../../Shared/Core/getDamageBonus";
```

Comment:
While correct, consider if using absolute paths wouldn't help readability (`Parser/Hunter/Shared/Core/getDamageBonus`), it's up to you. Same with the other module.

Reply:
![image](https://user-images.githubusercontent.com/29204244/31863870-1aba69ea-b754-11e7-8e75-a0ec2fc4a88c.png)

Reply:
Parser/**Hunter**/Shared/Core/getDamageBonus